src/main/java/net/dries007/tfc/world/settings/Settings.java

```diff
@@ -40,6 +41,7 @@ public record Settings(
         Codec.INT.fieldOf("rainfall_scale").forGetter(c -> c.rainfallScale),
         Codec.FLOAT.optionalFieldOf("rainfall_constant", 0f).forGetter(c -> c.rainfallConstant),
         RockLayerSettings.CODEC.fieldOf("rock_layer_settings").forGetter(c -> c.rockLayerSettings),
-        Codec.FLOAT.fieldOf("continentalness").forGetter(c -> c.continentalness)
+        Codec.FLOAT.fieldOf("continentalness").forGetter(c -> c.continentalness),
+        Codec.INT.optionalFieldOf("rock_layer_scale", 6).forGetter(c -> c.rockLayerScale) // match 1.18 scales
```

Comment:
pointless comment

Reply:
again, allergy to magic number with no context.